packages/sirius-web/frontend/sirius-web-application/src/views/edit-project/workbench-views/explorer/ExplorerView.tsx

```diff
@@ -41,12 +41,15 @@ import { GQLTreeEventPayload, GQLTreeRefreshedEventPayload } from './useExplorer
 
 const useStyles = makeStyles()((theme: Theme) => ({
   treeView: {
-    display: 'flex',
-    flexDirection: 'column',
+    display: 'grid',
+    gridTemplateColumns: 'auto',
+    gridTemplateRows: 'auto 1fr',
+    justifyItems: 'stretch',
+    overflow: 'auto',
+    flexGrow: 1,
```

Comment:
This kind of property is a major red flag just like `flex`, `flex-grow`, `margin`, `align-self`, etc. There are tons of examples. You cannot make any assumptions on the layout of the component that will reuse you. This would be a major coupling between `ExplorerView` and some internal details of `WorkbenchPart`.

There are multiple applications reusing Sirius Web that are integrating just some of our views in their own layout. You can't decide that you will take all the room available if your parent is using flexbox.